From 25604d238a82afd9775ecf7e562575fc7d053d73 Mon Sep 17 00:00:00 2001
From: Guy Nisan <gnisan@habana.ai>
Date: Wed, 01 Nov 2023 12:00:17 +0200
Subject: [PATCH] [SW-163765] gaudi2: change bis to rev3

When a device is bis complaint, change rev to 3, indicatin 225C device.

Change-Id: If7fb045d07c5e7bfbb60d50f6242f7df78d65cfb
Signed-off-by: Guy Nisan <gnisan@habana.ai>
---

diff --git a/arch/gaudi2/common/common.c b/arch/gaudi2/common/common.c
index 07fce4c..f4e91b3 100644
--- a/arch/gaudi2/common/common.c
+++ b/arch/gaudi2/common/common.c
@@ -505,7 +505,7 @@
 }
 
 #define HL_PCIE_REVISION_ID_MASK	0xFF
-#define HL225B_PCIE_REVISION_ID		0x2
+#define HL225C_PCIE_REVISION_ID		0x3
 static void preboot_pcie_device_id_vendor_id_set(void)
 {
 	u32 device_id_vendor_id = hl_pci_get_ven_id_dev_id();
@@ -525,7 +525,7 @@
 
 		/* clear low byte in PCIE class low word */
 		class_rev &= ~(HL_PCIE_REVISION_ID_MASK);
-		class_rev |= HL225B_PCIE_REVISION_ID;
+		class_rev |= HL225C_PCIE_REVISION_ID;
 
 		regmap_writel(class_rev, mmPCIE_DBI_CLASS_CODE_REVISION_ID);
 	}
